examples/py/cli.py

```diff
@@ -127,7 +127,7 @@ def print_usage():
 
     # unpack json objects (mostly for extra params)
     if arg[0] == '{' or arg[0] == '[':
-        args.append(exchange.unjson(arg))
+        args.append(json.dumps(arg))
```

Comment:
should be `reads` instead of `dumps` here, cause `unjson` is a parser

Reply:
ok